demo/demo.sh

```diff
@@ -25,8 +25,8 @@ run () {
         echo "$title '$t'; $(cprofile ${BASE_PATH}/$@) 2>&1 | tee ${BASE_PATH}/$t.log; sleep infinity"
 }
 
-run_router="$(run router soma-subscriptions-producer)"
```

Comment:
Please don't use `__init__.py` -- that means this code is parsed and loaded upon any load of the package, and any code in `__init__.py` other than imports is considered poor form by experienced Python users.

Please use `eosdatadistribution/tools/ostree_store.py` or so. Also, use `python -m eosdatadistribution.foo` instead of running scripts directly so that the search path is set up correctly.